src/Persistence/Models/ComponentDefinition.cs

```diff
@@ -12,28 +11,29 @@ namespace Microsoft.PowerPlatform.PowerApps.Persistence.Models;
 /// <summary>
 /// This class is meant to describe a Component definition.
 /// </summary>
-[FirstClass(templateName: BuiltInTemplates.Component)]
```

Comment:
why we're not considering this a First Class control type?

Reply:
Because they have custom Yaml converters